### PR TITLE
Create composer.json

### PR DESCRIPTION
**Pull request by PavelRykov**
Hi to all!
I would like to add this project into Packagist repository.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            "name": "Ewerton Assis",
+            "email": "earaujoassis@gmail.com",
+            "homepage": "https://earaujoassis.com/",
+            "role": "Developer"
+        },
+        {
+            "name": "Pavel Rykov",
+            "email": "p.rikov@drteam.rocks",
+            "homepage": "http://drteam.rocks/",
+            "role": "Maintainer"
```
Comment (earaujoassis) [inline composer.json]: I think "Contributor" would be more appropriate.

**Comment by PavelRykov** [comment]
@earaujoassis i agree with you, done

**Comment by earaujoassis** [comment]
This will be available in the next version, ok? I'll wait for #7 ! Thanks, @PavelRykov

**Comment by PavelRykov** [comment]
@earaujoassis can you create the packagist account, please? I want add you into package team.
Its need for "GitHub Service Hook" https://packagist.org/about#how-to-update-packages

By the way, packagist show me message: "There is no license information available for the latest version (dev-master) of this package.", maybe you can add the license line into composer.json? https://getcomposer.org/doc/04-schema.md#license

**Comment by earaujoassis** [comment]
@PavelRykov an account was created under the username `earaujoassis`. I'll update the `composer.json` file. Thanks.

**Comment by PavelRykov** [comment]
@earaujoassis thanks, i added you to members list, you should receive the message from packagist.

If you have a time, please set up "GitHub Service Hook", it's very easy, you just need open the Settings from project tabs, when webhooks page, then need add the packagist and set your API Token and Username. https://packagist.org/profile/

By the way, domain is not required.

**Pull request merged**